src/gentypes.jl

```diff
@@ -179,8 +178,8 @@ function addtype!(mod::ROSMsgModule, typ::String)
     if !(typ in mod.members)
         @debug("Message type import: ", _fullname(mod), ".", typ)
         if _nameconflicts(typ)
-            warn("Message type '$typ' conflicts with Julia builtin, ",
-                 "will be imported as '$(_jl_safe_name(typ,"Msg"))'")
+            @warn("Message type '$typ' conflicts with Julia builtin, ",
+                  "will be imported as '$(_jl_safe_name(typ,"Msg"))'")
```

Comment:
It looks like auto-concatenation of logging messages doesn't happen anymore and this ends up printing the substitution content verbatim in the resulting message. Can just use an extra call to `string` I suppose.